
# HG changeset patch
# User Thomas Jollans <thomas@jollans.com>
# Date 1276162427 -7200
# Node ID 04443c59a7a12bf517c9a5377ee4f4d654e6befb
# Parent  37bf7975fd062768b4f563713808785fc006cdd7
support Python 2.5 again (by not using the "numbers" module)

diff -r 37bf7975fd06 -r 04443c59a7a1 src/xmms2tray/__init__.py
--- a/src/xmms2tray/__init__.py	Sun Apr 11 23:34:32 2010 +0200
+++ b/src/xmms2tray/__init__.py	Thu Jun 10 11:33:47 2010 +0200
@@ -22,7 +22,6 @@
 import gettext
 from StringIO import StringIO
 import random
-import numbers
 
 # extra libraries
 import xmmsclient
@@ -124,7 +123,7 @@
     def newsong(self,res):
         v = res.value()
         
-        if isinstance(v, numbers.Integral):
+        if isinstance(v, (int, long)):
             # this is the ID. I want the whole info.
             self.xmms.medialib_get_info(res.value(),self.newsong)
         else:

